Openai-codex-harness-engineering-evals.md

```
@@ -0,0 +1,322 @@
+# OpenAI Codexにおけるハーネスエンジニアリング実践ガイド
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)